### PR TITLE
Metadata for GHC 9.6.5

### PR DESCRIPTION
**Pull request by wz1000**
https://gitlab.haskell.org/ghc/ghcup-ci/-/pipelines/93204

### CODE REVIEW

**Comment by wz1000** [comment]
https://github.com/haskell/ghcup-metadata/actions/runs/8700686971

**Comment by wz1000** [comment]
@hasufell do you know why jobs like rocky 8 etc. are failing? The logs seem to show everything worked.

**Comment by hasufell** [comment]
> @hasufell do you know why jobs like rocky 8 etc. are failing? The logs seem to show everything worked.

It probably means you're not distributing manpages for those platforms and the following line fails:

```
+ [[ -e /__w/_temp/foobarbaz/.ghcup/ghc/9.6.5/bin/../share/man/man1/ghc.1 ]]
```

**Comment by wz1000** [comment]
ok, that issue will not be fixed for this release.

**Comment by chreekat** [comment]
Are the man pages missing for all the 9.6 releases?

**Comment by hasufell** [comment]
> Are the man pages missing for all the 9.6 releases?

I believe pretty much every release is affected except 9.8.2, which I fixed manually.

File: ghcup-vanilla-0.0.7.yaml [hunk 1]
```diff
@@ -3931,6 +3931,104 @@ ghcupDownloads:
...
+      viChangeLog: https://downloads.haskell.org/~ghc/9.6.5/docs/users_guide/9.6.5-notes.html
+      viSourceDL:
+        dlHash: 87b389924f98c1a26c205122757338c8dab33ad1fcf670faa22622742432b93c
+        dlSubdir: ghc-9.6.5
+        dlUri: https://downloads.haskell.org/~ghc/9.6.5/ghc-9.6.5-src.tar.xz
+      viTags:
+      - base-4.18.2.1
+      viTestDL:
+        dlHash: dca3df48975cf7917db4e3ff6212e9478ce48337cb1a13e3007a3d1a8009a24c
+        dlSubdir: ghc-9.6.5
```
Comment (hasufell) [inline ghcup-vanilla-0.0.7.yaml:4030]: This is not correct

Comment (wz1000) [reply ghcup-vanilla-0.0.7.yaml]: fixed.

**Pull request merged**